pkg/metadata/finalizers.go

```diff
@@ -63,15 +63,14 @@ func HasInstanceFinalizer(obj metav1.Object, uid types.UID) bool {
 }
 
 // SetInstanceFinalizerUnstructured adds an instance-specific finalizer to an unstructured object.
-func SetInstanceFinalizerUnstructured(obj *unstructured.Unstructured, uid types.UID) error {
-	finalizerName := getInstanceFinalizerName(uid)
```

Comment:
Planning to remove the `getInstanceFinalizerName` too?

Reply:
removed along with 3 other non used functions